lesson05/shapes/circle_test.go

```diff
@@ -0,0 +1,150 @@
+package shapes
+
+import (
+	"math"
+	"testing"
+)
+
+func TestCircle_SetRadius(t *testing.T) {
+	type fields struct {
+		radius float64
+	}
+	type args struct {
+		r float64
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   bool
```

Comment:
If the return value is changed according to my comments above, the want value probably needs to be of type error.

Reply:
fixed